File: core/src/nl/t64/game/rpg/screens/inventory/tooltip/ShopSlotTooltipSell.java
package nl.t64.game.rpg.screens.inventory.tooltip;

import nl.t64.game.rpg.components.party.InventoryDescription;

import java.util.List;


public class ShopSlotTooltipSell extends ItemSlotTooltip {

    @Override
    void removeLeftUnnecessaryAttributes(List<InventoryDescription> descriptionList) {
        removeBuy(descriptionList);
    }

}

File: core/src/nl/t64/game/rpg/screens/inventory/tooltip/ShopSlotTooltipSell.kt
package nl.t64.game.rpg.screens.inventory.tooltip

import nl.t64.game.rpg.components.party.inventory.InventoryDescription


class ShopSlotTooltipSell : ItemSlotTooltip() {

    override fun removeLeftUnnecessaryAttributes(descriptionList: MutableList<InventoryDescription>) {
        removeBuy(descriptionList)
    }

}
